packages/core/src/account.ts

```diff
@@ -165,6 +175,12 @@ export class Account extends PBES2Container implements Storable {
         this.favorites.clear();
     }
 
+    /** Update account settings */
+    async setSettings(obj: Partial<AccountSettings>) {
+        Object.assign(this.settings, obj);
+        return this.settings;
+    }
```

Comment:
That method seems a bit superfluous. Is there any upside to this as opposed to setting the property directly?

Reply:
Good point, there isn't. I'll remove it.